client/my-sites/customer-home/cards/launchpad/keep-building.tsx

```diff
@@ -9,10 +9,14 @@ import { getSelectedSiteId } from 'calypso/state/ui/selectors';
 
 import './style.scss';
 
-function recordTaskClickTracksEvent( is_completed: boolean, task_id: string ) {
+const checklistSlug = 'keep-building';
+
+function recordTaskClickTracksEvent( task: Task ) {
 	recordTracksEvent( 'calypso_launchpad_task_clicked', {
-		task_id,
-		is_completed,
+		checklistSlug,
```

Comment:
Minor nit, looks like we're using underscores instead of camelCase for the other props, so this should be `checklist_slug` for consistency.